src/EnergyPlus/FluidProperties.cc

```diff
@@ -162,6 +162,9 @@ namespace FluidProperties {
         //   Array initializer only takes one argument.  std::bind is used to convert the
         //   actual initializer into a function of one argument.
 
+        if (state.dataFluidProps->GetFluidPropertiesData_called) return;
```

Comment:
Why is this necessary?

Reply:
For a simulation it's not. But I was worried that unit tests might call GetFluidPropertiesData more than once. If that happens you hit things like this:

    ++state.dataFluidProps->NumOfRefrigerants;